cromwell-logging/src/main/scala/cromwell/logging/BusinessLogging.scala

```diff
@@ -0,0 +1,105 @@
+package cromwell.logging
+
+import akka.actor.{ActorContext, Actor, ActorLogging, Props}
+import akka.util.Timeout
+import org.slf4j.{Logger, LoggerFactory}
+import scala.concurrent.duration._
+import cromwell.pubsub.{Subscriber, PubSubMediator}
+import scala.language.postfixOps
+
+/**
+  * Created by himanshu on 1/13/16.
+  */
+/**
+  * LogWrapper is used to realize different types of logger
+  * For example there is WorkflowLogger a specific implementation
+  * of LogWrapper.
+  */
+trait LogWrapper {
+  def debug(message : String)
+  def debug(message : String , t : Throwable)
+  def warn(message: String)
+  def warn(message: String, t : Throwable)
+  def error(message: String)
+  def error(message: String , t : Throwable)
+  def trace(message: String)
+  def trace(message: String, t : Throwable)
+  def info(message: String)
+  def info(message: String, t : Throwable)
+}
+
+/**
+  * Workflow execution event topics that business logging
+  * subscribed to and expecting,
+  */
+sealed trait WorkflowEvent
+case object CallExecutionEvent extends WorkflowEvent
+case object WorkflowExecutionEvent extends WorkflowEvent
+
+/**
+  * To report Business logging received as event from
+  * different components inside cromwell.
+  */
+trait BusinessLogEvent extends LogWrapper{
```

Comment:
I put this trait without sealed because to keep it open for other logging mechanism if needed, like you have WorkflowLogger, it may be useful to move it in Cromwell-logging package and utilize LogWrapper.